src/apiCalls.js

```diff
@@ -1,25 +1,41 @@
-// Think about setting all these with defaults in case there's something missing
-export const cleanParkData = (parks) => {
+const apiKey = process.env.REACT_APP_API_KEY;
+
+export const cleanParksData = (parks) => {
   const parksData = parks.data;
-  const cleanParks = parksData.map((park) => {
-    let cleanPark = {
-      id: park.id,
-      fullName: park.fullName ?? "No Name",
-      url: park.url || "No website",
-      description: park.description || "No description available",
-      directionsUrl: park.directionsUrl || "No directions available",
-      address: {
-        street: park.addresses?.[0]?.line1 || "No street address",
-        city: park.addresses?.[0]?.city || "No city",
-        stateCode: park.addresses?.[0]?.stateCode || "NA",
-        zip: park.addresses?.[0]?.postalCode || "00000",
-      },
-      images: park.images || [],
-      weatherInfo: park.weatherInfo || "Weather information not available",
-    };
-    // console.log(cleanPark);
-    return cleanPark;
+
```

Comment:
This looks great Logan! My only thought is maybe instead of accessing properties of the park object multiple times ('e.g., park. ....' ) you could use destructuring? Map and destructure? So you can just access them by their properties instead park.'propertyname' ? It might add some extra lines of code, so not sure if it's the most effective! But just a thought! 

.map((park) => {
  const {
    id,
    fullName = "No Name",
    url = "No website",
    description = "No description available",
    directionsUrl = "No directions available",
    addresses,
    images = [],
    weatherInfo = "Weather information not available",
  } = park;